src/mito2/src/read.rs

```diff
@@ -653,6 +672,7 @@ impl BatchBuilder {
 
         Ok(Batch {
             primary_key: self.primary_key,
+            pk_values: None,
```

Comment:
Can this value also be set on the write path? So that the inverted index can skip a decode operation.